components/CaseStatus/EditCaseStatusForm/EditCaseStatusForm.tsx

```diff
@@ -50,6 +60,10 @@ const EditCaseStatusForm: React.FC<{
           }
           if (field.id === 'endDate') {
             field.startDate = format(new Date(status.startDate), 'yyyy-MM-dd');
+
+            if (status.endDate) {
```

Comment:
Just wondering, would this code ever be triggered considering that when we end a case status, then the case status wouldn't be shown on the Details page. 
We wouldn't be able to get to the Edit page in this case.

Reply:
Hey Tony, this code in particular is to set the default value for the end date picker!

It won't change much if not setting the pre-filled value.